web/libs/ui/src/tokens/colors.scss

```
@@ -0,0 +1,99 @@
+:root {
+  --black-raw: 0 0 0;
+  --white-raw: 255 255 255;
+  --accent_color-raw: 86 111 207;
+  --danger_color-raw: 221 0 0;
+  --danger_color_dimmed: rgb(var(--danger_color-raw) / 50%);
+  --black: rgb(var(--black-raw));
+  --white: rgb(var(--white-raw));
+  --black_2:  rgb(var(--black-raw) / 2%);
+  --black_4:  rgb(var(--black-raw) / 4%);
+  --black_5:  rgb(var(--black-raw) / 5%);
+  --black_10: rgb(var(--black-raw) / 10%);
+  --black_15: rgb(var(--black-raw) / 15%);
+  --black_20: rgb(var(--black-raw) / 20%);
+  --black_40: rgb(var(--black-raw) / 40%);
+  --sand_0: #FDFDFC;
+  --sand_100: #F9F8F6;
+  --sand_200: #F0EFEB;
+  --sand_300: #E1DED5;
+  --sand_400: #CAC5B8;
+  --sand_500: #A49F95;
+  --sand_600: #6B6860;
+  --sand_700: #45433E;
+  --sand_800: #262522;
+  --sand_900: #12110D;
+  --plum_500: #CC6FBE;
+  --plum_400: #E37BD3;
+  --plum_200: #F1BDE9;
+  --plum_100: #F7D6F2;
+  --plum_0: #FBF2FC;
+  --persimmon_0: #FFF1EE;
+  --persimmon_300: #FF9F89;
+  --persimmon_400: #FF7557;
+  --canteloupe_0: #FFF6EF;
+  --canteloupe_100: #FFE4D0;
+  --canteloupe_400: #FFA663;
+  --canteloupe_500: #E69559;
+  --canteloupe_600: #CC854F;
+  --canteloupe_700: #B37445;
+  --kale_0: #F4F9F9;
+  --kale_300: #7ACEC1;
+  --kale_400: #57B7AB;
+  --red_1: #FFF1F0;
+  --red_3: #FFCCC7;
+  --red_7: #CF1322;
+  --red_10: #5C0011;
+  --grape_0: #F0F3FE;
+  --grape_100: #D4DBFB;
+  --grape_500-raw: 86, 111, 207;
+  --grape_500: rgb(var(--grape_500-raw));
+  --grape_600: #576CC1;
+  --grape_700: #4C5FA9;
+  --grape_800: #37447A;
+  --primary_link: var(--grape_500);
```

```suggestion
  --grape_800: #37447A;

  --primary_link: var(--grape_500);
```
let's add a separation or maybe even a comment to distinguish low level colors and semantic names. unless this file is autogenerated